cloud_functions/climateiq_export_to_aws_cf/main.py

```
@@ -0,0 +1,72 @@
+from datetime import datetime
+import os
+
+import boto3
+import flask
+import functions_framework
+from google.cloud import firestore, storage
+
+# GCS bucket name, where merged prediction outputs are stored.
+GCS_BUCKET_NAME = (
+    os.environ.get("BUCKET_PREFIX", "") + "climateiq-spatialized-merged-predictions"
+)
+# AWS bucket name, where to copy files to.
+S3_BUCKET_NAME = "climateiq-data-delivery"
+
+# IDs for Firestore.
+SECRET_KEYS_COLLECTION_ID = "secret_keys"
+AWS_KEYS_DOC_ID = "aws_keys"
+AWS_ACCESS_KEY_ID_KEY = "aws_access_key_id"
+AWS_SECRET_ACCESS_KEY_KEY = "aws_secret_access_key"
+
+
+@functions_framework.http
+def export_to_aws(request: flask.Request) -> tuple[str, int]:
+    try:
+        prefix = _get_prefix_id(request)
+    except ValueError as e:
+        return (str(e), 400)
+
+    storage_client = storage.Client()
+    blobs_to_export = storage_client.list_blobs(GCS_BUCKET_NAME, prefix=prefix)
```

lets return an error if no blobs exist with the given prefix